src/test/java/com/coreos/jetcd/internal/impl/LoadBalancerTest.java

```diff
@@ -0,0 +1,76 @@
+package com.coreos.jetcd.internal.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.coreos.jetcd.Client;
+import com.coreos.jetcd.ClientBuilder;
+import com.coreos.jetcd.KV;
+import com.coreos.jetcd.kv.PutResponse;
+import io.grpc.PickFirstBalancerFactory;
+import io.grpc.util.RoundRobinLoadBalancerFactory;
+import org.testng.annotations.Test;
```

Comment:
let's have a different Pr to clean up old tests from testNg to junit if we have time.